devices.js

```diff
@@ -942,6 +967,15 @@ const devices = [
         fromZigbee: [fz.xiaomi_battery_3v, fz.DJT11LM_vibration],
         toZigbee: [tz.DJT11LM_vibration_sensitivity],
     },
+    {
+        zigbeeModel: ['lumi.vibration.agl01'],
+        model: 'DJT12LM',
+        vendor: 'Xiaomi',
+        description: 'Aqara vibration sensor',
+        supports: 'drop, tilt and touch',
+        fromZigbee: [fz.genOnOff_cmdOn],
```

Comment:
returning `click: on` makes no sense for this device, does `fz.DJT11LM_vibration` work?

Reply:
by default  it returns click: on once the device reveives vibration three times in a  short time only and  fz.DJT11LM_vibration will not work(but can be configured by using aqara's new gateway and their Aqara home app ). And aqara said they will not opensource shortly and the samples i got were only engineer samples.  Currently I don't have enough time to figure out

Reply:
In that case return `{"action": "vibration"}`, that makes much more sense then click: on.